components/sendgrid/actions/send-email-single-recipient/send-email-single-recipient.js

```diff
@@ -0,0 +1,325 @@
+      optional: true,
+    },
+    categories: {
+      type: "string[]",
+      label: "Categories",
+      description:
+        "A string array of category names for this message. Each category name may not exceed 255 characters. Example: `[\"category1\",\"category2\"]`",
+      optional: true,
+    },
+    customArgs: {
+      type: "string",
+      label: "Custom Args",
+      description:
+        "Values that are specific to the entire send that will be carried along with the email and its activity data. Key/value pairs must be strings. Substitutions will not be made on custom arguments, so any string that is entered into this parameter will be assumed to be the custom argument that you would like to be used. This parameter is overridden by `custom_args` set at the personalizations level. Total `custom_args` size may not exceed 10,000 bytes.",
+      optional: true,
+    },
+    sendAt: {
+      type: "integer",
+      label: "Send At",
+      description:
+        "A unix timestamp allowing you to specify when you want your email to be delivered. This may be overridden by the `send_at` parameter set at the personalizations level. Delivery cannot be scheduled more than 72 hours in advance. If you have the flexibility, it's better to schedule mail for off-peak times. Most emails are scheduled and sent at the top of the hour or half hour. Scheduling email to avoid peak times — for example, scheduling at 10:53 — can result in lower deferral rates due to the reduced traffic during off-peak times.",
+      optional: true,
+    },
+    batchId: {
+      type: "integer",
+      label: "Batch Id",
+      description:
+        "An ID representing a batch of emails to be sent at the same time. Including a batch_id in your request allows you include this email in that batch. It also enables you to cancel or pause the delivery of that batch. For more information, see the [Cancel Scheduled Sends API](https://sendgrid.com/docs/api-reference/).",
+      optional: true,
+    },
+    asm: {
+      type: "object",
+      label: "ASM",
+      description:
+        "Advanced Suppression Manager. An object allowing you to specify how to handle unsubscribes.",
+      optional: true,
+    },
+    ipPoolName: {
+      type: "string",
+      label: "Ip Pool Name",
+      description: "The IP Pool that you would like to send this email from.",
+      optional: true,
+    },
+    mailSettings: {
+      type: "object",
+      label: "Mail Settings",
+      description:
+        "A collection of different mail settings that you can use to specify how you would like this email to be handled.",
+      optional: true,
+    },
+    trackingSettings: {
+      type: "object",
+      label: "Tracking Settings",
+      description:
+        "Settings to determine how you would like to track the metrics of how your recipients interact with your email.",
+      optional: true,
+    },
+  },
+  methods: {
+    ...common.methods,
+  },
+  async run() {
```

Comment:
Include the `$` variable.
```suggestion
  async run({ $ }) {
```